src/pages/Home/ReservesRanges/ReservesRanges.tsx

```diff
@@ -1,33 +1,216 @@
 import React from 'react'
-import { Grid, Typography } from '@mui/material'
+import { Grid, Typography, Box, MenuItem, TextField, TextFieldProps, InputAdornment, Button } from '@mui/material'
 import { COLORS } from '@/consts/colors.ts'
-import DateRange from '@/pages/Home/ReservesRanges/DateRange/DateRange.tsx'
 import { IHomeChoiceFormField } from '@/pages/Home/HomeChoiceFormField/HomeChoiceFormField.tsx'
+import { HOME_OPTIONS_MAP } from '@/pages/Home/Home.consts.ts'
+import { IFormikControllers } from '../Home.module'
+import { DatePickerProps, MobileDatePicker, PickersDay } from '@mui/x-date-pickers'
+import CalendarTodayIcon from '@/icons/calendar_today'
+import dayjs from 'dayjs'
+import PlusIcon from '@/icons/plus_icon'
 
 interface IReservesRanges {
   startDateProps: IHomeChoiceFormField
   endDateProps: IHomeChoiceFormField
+  startDateProps2: IHomeChoiceFormField
+  endDateProps2: IHomeChoiceFormField
+  startDateProps3: IHomeChoiceFormField
+  endDateProps3: IHomeChoiceFormField
   recruitmentTypeProps: IHomeChoiceFormField
+  recruitmentTypeProps2: IHomeChoiceFormField
+  recruitmentTypeProps3: IHomeChoiceFormField
 }
 
-const ReservesRanges: React.FC<IReservesRanges> = ({ startDateProps, endDateProps, recruitmentTypeProps }) => {
+
+type DateRangePickerValueType = {
+  start: unknown;
+  end: unknown;
+};
+
+interface DateRangePickerProps
+  extends Omit<DatePickerProps<unknown, unknown>, 'value'> {
+  value: DateRangePickerValueType | null;
+}
+
+const DateRangePicker = (props: DateRangePickerProps) => {
```

Comment:
Can you please extract this component to a separate file?

Reply:
@EfratGutman ,  done